spec/features/gardens/new_spec.rb

```diff
@@ -33,14 +27,42 @@
       expect(page).to have_button('Create Garden')
     end
 
-    it 'fills new garden form, submits and is redirect back to dashboard' do
+    it 'fills new garden form, submits and is redirected to garden index' do
+      name = 'The Grove'
+      longitude = 25.0000
+      latitude = 71.0000
+      description = 'My first garden'
+
+      expected_output = File.read('spec/fixtures/new_garden.json')
+
+      stub_request(:post, "https://solar-garden-be.herokuapp.com/api/v1/gardens?description=#{description}&latitude=#{latitude}&longitude=#{longitude}&name=#{name}&private=false&user_id=#{@user.id}").to_return(status: 200, body: expected_output, headers: {})
+
+      stub_request(:get, "https://solar-garden-be.herokuapp.com/api/v1/gardens/50").to_return(status: 200, body: expected_output, headers: {})
```

Comment:
To me, this feels like we are stubbing the things that we are testing. I think this test would not break if we changed the requirements to create a garden. For example, if we said a garden needed an attribute for `hours of operation`, this test would still pass even if we didn't pass that required attribute. Correct?

If the above is the case I don't see the full value of having this test. Would love to hear any thoughts on that. How could we break this test?

Reply:
Yes I have the same concerns. It feels like I'm just testing the variables within the test and not the actual functionality. I am not sure how to actually make this a test. I was double checking by also confirming in local host. Maybe we can merge as is since Ian recommended this method and bring this up as a topic with him because it'll impact all of our create, update, destroy tests? Maybe this is why they say testing FE is so tricky 🤷